spec/integration/primitives_spec.rb

```diff
@@ -18,12 +18,12 @@
           },
         }
 
-      let(:iso_date) { Time.now.iso8601.gsub(':', '') }
```

Comment:
My bad, it is not the length, but illegal symbol (`+` for timezone):
```
       expected no Exception, got #<Docker::Error::ServerError: machine_image[docker_image_spec_2015-06-23T213107+0300] (basic_chef_client::block line 39) had an error: Docker::Error::ServerError: Illegal tag name (docker_image_spec_2015-06-23T213107+0300): only [A-Za-z0-9_.-] are allowed, minimum 1, maximum 128 in length
```
Still don't get, why didn't you get that error.
```
$ docker version
Client version: 1.6.2
Client API version: 1.18
Go version (client): go1.4.2
Git commit (client): 7c8fca2
OS/Arch (client): linux/amd64
Server version: 1.6.2
Server API version: 1.18
Go version (server): go1.4.2
Git commit (server): 7c8fca2
OS/Arch (server): linux/amd64

$ ruby --version
ruby 2.1.5p273 (2014-11-13) [x86_64-linux-gnu]
```